EIPS/eip-5585.md

```diff
@@ -163,7 +159,7 @@ This standard is compatible with [EIP-721](./eip-721.md) since it is an extensio
 
 ## Security Considerations
 
-If someone buys an NFT within the duration of an authorization, they will not have to stake anything, providing no incentive to cancel the authorization.
+If a NFT owner sold his NFT with an authorization, we need to make sure the new onwer can have the rest authorization fee and will not cancel the authorization.
```

Comment:
Not sure about this wording. Might be worth rephrasing.

Reply:
I have rephrased those words, thanks @Pandapip1